src/Domain.jl

```diff
@@ -152,7 +152,7 @@ function update_params!(d::Domain, params::Union{AbstractVector,DataFrameRow})::
         end
     end
 
-    to_floor = (p_df.ptype .== "integer")
+    to_floor = (p_df.ptype .== "integer") .| (p_df.ptype .== "categorical")
```

Comment:
I suggest we define a constant that specifies `ptype`s that are discrete, and write a function that does this check as it is a common task that is repeated.

Reply:
Should this constant/function be in Domain.jl?